src/routes/(admin)/tags/[type]/[id]/+page.server.ts

```
@@ -0,0 +1,174 @@
+import type { PageServerLoad, Actions } from './$types.js';
+import { tagSchema, tagTypeSchema } from './schema';
+import { superValidate } from 'sveltekit-superforms';
+import { zod } from 'sveltekit-superforms/adapters';
+import { getSessionTokenCookie } from '$lib/server/auth.js';
+import { fail } from '@sveltejs/kit';
+import { BACKEND_URL } from '$env/static/private';
+import { redirect, setFlash } from 'sveltekit-flash-message/server';
+
+export const load: PageServerLoad = async ({ params, cookies }: any) => {
```

ESLint

[error] 'cookies' is defined but never used.

(@typescript-eslint/no-unused-vars)